assets/xml/objects/object_mm.xml

```diff
@@ -1,68 +1,80 @@
 ﻿<Root>
+    <!-- Assets for the Postman -->
```

Comment:
Something to take in consideration that we are trying to get everybody who has contributed / wants to contribute to MM decomp to agree to licensing the repo before merging PRs from people who has not yet agreed to it. It is better explained here: https://github.com/louist103/mm-licensing/issues/1.
Could you please read about it and post in that issue if you agree or not. Thanks!